ddl/ddl_worker.go

```diff
@@ -471,6 +472,22 @@ func chooseLeaseTime(t, max time.Duration) time.Duration {
 
 // runDDLJob runs a DDL job. It returns the current schema version in this transaction and the error.
 func (w *worker) runDDLJob(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error) {
+	defer func() {
+		r := recover()
```

Comment:
We have added `recover` to the worker's `start`. Is it repeated？

Reply:
not enough, what if all worker panic?

Reply:
No reapeat，`recover` in `start` will cause ddl worker already  exit, and then there is no ddl worker in this tidb server.